plugin/route53/README.md

```diff
@@ -57,6 +57,12 @@ route53 [ZONE:HOSTED_ZONE_ID...] {
 
 *   **DURATION** A duration string. Defaults to `1m`. If units are unspecified, seconds are assumed.
 
+*   `alias_refresh` can be used to control how often do we refresh the aliases from Route 53. Each alias
```

Comment:
on a first impression, this looks odd that it needs to be called out explicitly.

What's a standalone test you can add that fails with the current impl. but works with your pr?

Reply:
The TTL of alias in route53 follows the source TTL. From that standpoint I think an alias_refresh is not needed. Instead the refresh interval could be based on the TTL of the source?

Reply:
@yongtang I don't mind remove alias_refresh at all, the reason it's there is that the health-checks can be 15 seconds or 30 seconds when they are defined, so this allows you to override the refreshing period. As for the source TTL, there is no way to change it other than leave it at 60 seconds. But mind you this is valid for all records not just aliases. Should I remove them @yongtang ?

@miekg Any record returned by AWS that is an ALIAS will fail, 
https://github.com/coredns/coredns/pull/3442/files#diff-60d1faf3afabbffbb6c6dd8dc9e5d95bR64-R67
This will never work as AWS will never return an IP address when you get the AWS ZONE from them, it will only return a record that says it's an alias.

Reply:
I've added a same zone lookup, but regarding the timing how do you want to resolve it? @yongtang any ideas? We can make it dynamic, based on the record TTL but that will get unnecessary complicated, if you ask me. But I'm open to any ideas. I still feel that having it configurable would offer the simplest and least complicated option.

@miekg this is what we get from the AWS API when we process the alias records.

```
(*route53.ResourceRecordSet)(0xc000746180)({
  AliasTarget: {
    DNSName: "d21v7mkn0v2jz6.cloudfront.net.",
    EvaluateTargetHealth: false,
    HostedZoneId: "Z2FDTNDATAQYW2"
  },
  Name: "files.msp.sh.",
  Type: "A"
})
````

Reply:
alias_refreshed needs to be removed - this doesn't warrant more config knobs

Reply:
removed.